flow-typed/types/omid-access-modes-config.js

```diff
@@ -0,0 +1,29 @@
+// @flow
+
+/**
+ * {@link https://developers.google.com/interactive-media-ads/docs/sdks/html5/client-side/omsdk#access_modes different access modes for verification scripts}
+ * @typedef AccessMode
+ * @type { 'FULL' | 'CREATIVE' | 'DOMAIN' | 'LIMITED'}
+ */
+declare type AccessMode = 'FULL' | 'CREATIVE' | 'DOMAIN' | 'LIMITED';
+
+/**
+ * {@link https://developers.google.com/interactive-media-ads/docs/sdks/html5/client-side/reference/js/google.ima#.OmidVerificationVendor different verification script providers}
+ * @typedef OmidVendor
+ * @type { 'OTHER' | 'MOAT' | 'DOUBLEVERIFY' | 'INTEGRAL_AD_SCIENCE'  | 'PIXELATE'  | 'NIELSEN'  | 'COMSCORE' | 'MEETRICS' | 'GOOGLE'}
+ */
+declare type OmidVendor = 'OTHER' | 'MOAT' | 'DOUBLEVERIFY' | 'INTEGRAL_AD_SCIENCE'  | 'PIXELATE'  | 'NIELSEN'  | 'COMSCORE' | 'MEETRICS' | 'GOOGLE';
```

Comment:
I would keep it as string and just refer to the list in ima docs. so we won't need to change it on ima updates.
same for AccessMode